Taarafo.Core/Models/Events/Event.cs

```diff
@@ -12,12 +12,12 @@ public class Event
 		public Guid Id { get; set; }
 		public string Name { get; set; }
 		public string Description { get; set; }
-		public DateTime Date { get; set; }
+		public DateTimeOffset Date { get; set; }
```

Comment:
Changing the type here, will need a migration to adjust the Event table.
Like wise with the other columns.